docs/user-guide/auto_sync.md

```diff
@@ -75,6 +75,10 @@ spec:
       selfHeal: true
 ```
 
+NOTE: Disabling self-heal doesn't guarantee that live cluster changes won't be reverted for multi-source apps even if the
+resource's source didn't change, since other source may change and trigger autosync. Disable autosync if needed to
+handle these cases.
```

Comment:
@andrii-korotkov-verkada , thanks for adding the note. What do you think about wording the same as below?

```suggestion
Disabling self-heal does not guarantee that live cluster changes won't be reverted in multi-source applications. Even if a resource's source remains unchanged, changes in one of the sources can trigger `autosync`. To handle such cases, consider disabling `autosync`.
```

Reply:
Sure, updated.